care/facility/api/serializers/patient.py

```diff
@@ -210,6 +210,15 @@ def validate_year_of_birth(self, value):
             raise serializers.ValidationError(msg)
         return value
 
+    def validate_name(self, value):
+        if value is None:
+            raise serializers.ValidationError("Patient name is required.")
+        if not value.strip():
+            raise serializers.ValidationError("Patient name cannot be empty.")
```

Comment:
I think the model will handle this by default, no need to add these validations in the serializer.